### ISSUE TITLE
Feature: Duplication of invoices

### ISSUE THREAD

**Issue by mint-dewit** [madisvain/upcount#68]
I often create very similar invoices every month (only the amounts, dates and number differ) so it is nice to be able to quickly copy invoices.

I've created this in a branch on my fork already (that I use to be able to generate custom style invoices) and would be open to working towards merging this. It would require a little bit of work on my end (as it's based on automatic incrementing of invoice numbers, more on that in #32) so I'd like to poll interest first.

**Comment by madisvain** [comment]
I totally agree on this. The option to duplicate an invoice with auto incremental invoice numbers is definitely something that would help a lot with not having to think about what the next number needs to be an also usually you are billing the same kind of works every month so filling all the data in again is quite tedious work.

I'd be happy to take a look at your fork and a pull-request if possible.

**Comment by madisvain** [comment]
@baltedewit this pretty much sums up whats needed yes.
https://github.com/baltedewit/upcount/commit/e1d6e5275f1027d258f90623ccd10b625500b5f3

Are you interested in making this into a pull-request if you are taking part if [Hacktoberfest](https://hacktoberfest.digitalocean.com/)?

There are two potential pull-requests from you which I'd be happy to review & merge. Duplication and number automation.
